lib/src/pkg_resolution.dart

```diff
@@ -308,6 +308,12 @@ class PkgDependency extends Object
   bool get isLatest => available == null;
   bool get isOutdated => !isLatest;
 
+  bool get isHosted =>
```

Comment:
Is an 'inherited' constraint always hosted?

Reply:
Good point, I'm not sure about it.
I'd go with these for now, and run some checks on the pub site to see how they pan out.